README.md

```diff
@@ -21,27 +21,27 @@ Consider this file name:
 What is the purpose of this file, and what is the purpose of the numbers at the beginning of its name?
 
 ```text
-Your answer...
+This is a migration file. The purpose of migration file is to create the schema with associated field inputs for the particular class.
 ```
 
 ### Question 3
 
 In a Rails application, how is the router related to controller actions?  
 
 ```text
-Your answer...
+Router determines which controller methods are required.
 ```
```

Comment:
👍 A router sees if a request made to our application has a route defined for it, if it does, it will map the request to the appropriate controller and action...